docs/zkapps/tutorials/04-zkapp-ui-with-react.mdx

```diff
@@ -142,27 +142,6 @@ For this tutorial, you run commands from the root of the `04-zkapp-browser-ui/ui
 
 Each time you make updates, then build or deploy, the TypeScript code is compiled into JavaScript in the `build` directory.
 
-### Install the dependencies
```

Comment:
You are correct. It is fine to remove this, but I would prefer to keep the section describing the sub-directories and remove the section about running `npm install`.  For example. 
### Install the dependencies

When you ran the `zk project` command, your UI was created in the project directory: `/04-zkapp-browser-ui/ui` the project has two sub-directories:

  - `contracts`: The smart contract code 
  - `ui`: Where you write your UI code

The dependancies in each sub-directory are installed automatically by the `zkApp-cli`.

Reply:
done